docs/content/en/docs/getting-started/production-environment/snow-getstarted.md

```diff
@@ -0,0 +1,227 @@
+* To be run on an Admin instance in a Snowball Edge device. See [Initial Device Setup](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) for setting up the devices, launching the Admin instance, fetching and copying the device credentials to the admin instance for `eksctl` CLI to consume.
+
+Also, see the [Ports and protocols]({{< relref "/docs/reference/ports.md" >}}) page for information on ports that need to be accessible from control plane, worker, and Admin machines.
+
+## Steps
+
+The following steps are divided into two sections:
+
+* Create an initial cluster (used as a management or standalone cluster)
+* Create zero or more workload clusters from the management cluster
+
+### Create an initial cluster
+
+Follow these steps to create an EKS Anywhere cluster that can be used either as a management cluster or as a standalone cluster (for running workloads itself).
+
+<!-- this content needs to be indented so the numbers are automatically incremented -->
+1. Set an environment variables for your cluster name
+   
+   ```bash
+   export CLUSTER_NAME=mgmt
+   ```
+
+1. Generate a cluster config file for your Snow provider
+
+   ```bash
+   eksctl anywhere generate clusterconfig $CLUSTER_NAME --provider snow > eksa-mgmt-cluster.yaml
+   ```
+
+1. Optionally import images to private registry
+   
+   This optional step imports EKS Anywhere artifacts and release bundle to a local registry. This is required for air-gapped installation.
+   
+   * [Configure Private Registry on Snowball](https://docs.aws.amazon.com/snowball/latest/ug/whatissnowball.html) shows an AWS sample of selecting and building a private registry in a Snowball Edge device.
+   * For air-gapped scenario, run the `import images` with `--input` and `--bundles` arguments pointing to the artifacts and bundle release files that pre-exist in the admin instance.
+   * Refer to the [Registry Mirror configuration]({{< relref "../../reference/clusterspec/optional/registrymirror" >}}) for more information about using private registry.
+
+   ```bash
+   eksctl anywhere import images \
+      --input /usr/lib/eks-a/artifacts/artifacts.tar.gz \
+      --bundles /usr/lib/eks-a/manifests/bundle-release.yaml \
+      --registry $PRIVATE_REGISTRY_ENDPOINT \
+      --insecure=true
+   ```
+
+1. Modify the cluster config (`eksa-mgmt-cluster.yaml`) as follows:
+   * Refer to the [Snow configuration]({{< relref "../../reference/clusterspec/snow" >}}) for information on configuring this cluster config for a Snow provider.
+   * Add [Optional]({{< relref "/docs/reference/clusterspec/optional/" >}}) configuration settings as needed.
+
+1. Set License Environment Variable
+
+   If you are creating a licensed cluster, set and export the license variable (see [License cluster]({{< relref "/docs/tasks/cluster/cluster-license" >}}) if you are licensing an existing cluster):
+
+   ```bash
+   export EKSA_LICENSE='my-license-here'
+   ```
+
+1. Configure Curated Packages
```

Comment:
As you mentioned earlier, we'll need to note how curated package consumption may be different (or not supported) for air-gapped installations.

Reply:
Good. Thanks.